package.json

```diff
@@ -53,7 +53,7 @@
     "@storybook/addons": "^5.3.9",
     "@storybook/react-native": "^5.3.21",
     "@testing-library/jest-native": "^3.1.0",
-    "@testing-library/react-hooks": "3.2.1",
+    "@testing-library/react-hooks": "3.4.2",
```

Comment:
We'll needa verify that existing tests that use `rerender` do not depend on the new prop argument incorrectly too

Reply:
I have refactored all `rerender` to not include `props`.

Moving forward, we can simply reassign the `props` used by the hook, then call `rerender` to call the hook with the new `props`.

```typescript
let ids = ["ID1"];
const { rerender, result } = renderHook(
  () => useCart(ids, ...),
  { wrapper }
)

// Say we want to update ids in useCart hook
ids = ["ID1", "ID2"];
rerender();
```

The `props` in `rerender` will now be used to update the `wrapper` context values.
```typescript
let ids = ["ID1"];
const { rerender, result } = renderHook(
  () => useCart(ids, ...),
  { wrapper }
)

// Say we want to update the context in wrapper
rerender({ products: newProducts });
```